tests/xtro-sharpie/tvOS-VideoToolbox.ignore

```diff
@@ -0,0 +1,2 @@
+!missing-field! kVTVideoDecoderSpecification_PreferredDecoderGPURegistryID not bound
```

Comment:
minor: `git blame ` can help but it's easier (in `.ignore` files) when a comment like `### API not marked with unavailable (but without availability for tvOS/watchOS)` is added